examples/kubernetes-zones/README.md

```diff
@@ -0,0 +1,74 @@
+# Availability Zones
+
+To protect your cluster from datacenter-level failures, you can enable the Availability Zones feature for your cluster by configuring `"availabilityZones"` for the master profile and all of the agentPool profiles in the cluster definition. 
+
+ - This feature only applies to Kubernetes clusters version 1.12+. 
+ - Supported values are arrays of strings, each representing a supported availability zone in a region for your subscription. e.g. `"availabilityZones": ["1","2"]` represents zone 1 and zone 2 can be used. 
```

Comment:
Maybe reword the second sentence like this:

For example, \`"availabilityZones": ["1","2"]\` indicates zone 1 and zone 2 can be used.